week5/TaskApp/TaskApp/View/TaskListView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TaskListView.swift
+//  TaskApp
+//
+//  Created by Melissa Zellhuber on 10/01/24.
+//
+
+import SwiftUI
+
+struct TaskListView: View {
+    @StateObject var store = TaskStore()
```

Comment:
It's a good practice to make `@StateObject` wrapped property also a private one. This way compiler can prevent accidental setting of the property from outside of a container. For more details, please check out the official page https://developer.apple.com/documentation/swiftui/stateobject